worker/convert/converter.py

```diff
@@ -1,10 +1,39 @@
 import logging
 
 from PIL import Image
+from wand.image import Image as Img
```

Comment:
And dieser Stelle ist es sinnvoll beide Image Module unter aussagekräftigen Namen zu importieren.